src/Components/Publishing/Sections/ArtworkCaption.tsx

```diff
@@ -254,10 +220,13 @@ const ArtistName = styled.span`
     .artist-name {
       margin-right: 30px;
     }
-  `}
+  `};
 `
 
-const div: StyledFunction<StyledArtworkCaptionProps & React.HTMLProps<HTMLDivElement>> = styled.div
+const div: StyledFunction<
+  StyledArtworkCaptionProps & React.HTMLProps<HTMLDivElement>
+> =
+  styled.div
```

Comment:
@kanaabe - with latest TypeScript version this form of typing passed props to styled-components can be way simplified: 

```typescript
interface Props {
  sectionLayout: string
}

const ArtworkCaption = styled.div`
  padding: ${(props: Props) => (props.sectionLayout === "fillwidth" ? "0 10px;" : "0;")}
`
```

Reply:
Yea, I'm so excited for this! Gonna merge tho since this diff is from Prettier and not a code change. Thanks for bringing it up!